packages/cli/test/utils/simulation/cl_clients/lodestar.ts

```diff
@@ -52,6 +52,7 @@ export const generateLodestarBeaconNode: CLClientGenerator<CLClient.Lodestar> =
     "network.allowPublishToZeroPeers": false,
     discv5: true,
     "network.connectToDiscv5Bootnodes": true,
+    "network.rateLimitMultiplier": 0,
```

Comment:
It turned out that with recent changes if there is rate limit error, the peer get disconnected. Had to disable the limit for sim tests.